spec/support/form_helpers.rb

```diff
@@ -21,15 +21,16 @@ def fill_in_activity_form(
     flow: "ODA",
     finance: "Standard grant",
     aid_type: "General budget support",
-    tied_status: "Untied"
+    tied_status: "Untied",
+    level: nil
```

Comment:
I think it would be nice to have a default for the level. If one calls `fill_in_activity_form` without passing in a level param, the result of line 33 is `"Purpose of "`. The test still passes but it's a bit odd. As all the other params have a default value, I think it's fine to add one to level too.

Reply:
It is a good point. I added the level param in every `fill_in_activity_form` from the specs files to make sure they all pass a parameter (fund, programme or project) to the form_helpers. But I don't see any harm in adding a default value here. Should we use the generic `activity` as default value?

Reply:
I think use `fund` as the default as it's the top level activity type, but it doesn't matter particularly! I don't think `activity` is a level though.

Reply:
True. Activity is not a level. I'll use `fund`. Thank you Laura!